pinc/showstartexts.inc

```diff
@@ -27,8 +27,9 @@ function showstartexts($etext_limit,$type) {
     echo "<div class='star-text-summary'>";
 
     echo "<div class='star-header'>";
-    // Display star graphic
-    echo "<img src='$code_url/graphics/{$type}_star.jpg' height='38' width='40' alt='$type star'><br>\n";
+
+    // Display star
+    echo "<div class='star-$type'>★</div>";
```

Comment:
We're losing some accessibility without the alt text. A styled star may not read well with a screen reader. Perhaps add aria img role with aria label
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img

Reply:
Or there could be a title with text as alt was. But it should be translated.

Reply:
In this particular case, would that be value-added?

Mind you, I'm not against accessibility accommodations, but the stars are essentially visual attention-getters. No useful information is missed by leaving out alt text that says "Gold star", "Silver star", or "Bronze star".

Reply:
Take a look at this screenshot simulating color blindness:
![image](https://user-images.githubusercontent.com/1526781/94302571-0aa11d00-ff32-11ea-9f95-f122abdb6ad2.png)
We do explain the sections underneath the star, but if they were using a screen reader, it may help to know what star they are on before having the screen reader read the text below it. Now i know the site probably fails in a million ways accessibility related, but we had alt text before so in this case, we are making it worse.

Reply:
Fair point. And I was thinking primarily of the linked pages, not at the blurbs down at the bottom of the home page. Definitely more useful on the home page, whereas the stars are being added to the linked pages.

Reply:
All good points - thank you for catching this, @chrismiceli.

I've added an `aria-label` to the divs.

Reply:
Thanks